libmscore/sym.cpp

```diff
@@ -4354,7 +4354,7 @@ QVector<QString> Sym::symUserNames = {
       "Finger",
       "Fingernails",
       "Fist",
-      "Guiro scraper",
+      u8"Güiro scraper",
```

Comment:
I wouldn't change this one. It comes directly from SMuFL and there is no ü there. It would be overwritten as soon as we update to a new version of SMuFL...